### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by mannnyyy** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#65] [Ideas]
Hi David,

Do you plan to make AlDente available to M1 ARM Macs ? It would be great.

Thanks.

Manuel

**Comment by SvenderCell** [comment]
Yes, I also have an M1 processor now. This is my first Mac I bought and I'd like to be able to extend its life as long as possible. So it would be great that when I do work at home, I can power the Laptop with the adapter and let the battery be in peace at a good health like 50% or 70%. I know I am just repeating what everyone knows here in this forum and that it is exactly what the app does for intel processors. So, I hope as well that the M1-AIDente is coming out soon :-)

Best regards,

Sven

**Comment by MatthiasKerbl** [comment]
The time has come: AlDente for Apple Silicon (M1 MacBooks) is out now. We thank everyone who has helped us here in the forum and hope you like it. That's not all, we have also started working on a pro version of AlDente with advanced features several weeks ago. Since we have put a lot of hours into developing the app, it will cost a little. However, there will always be this free and open-source version of AlDente updated with the new M1 SMC keys.  If you have any feature requests or ideas for AlDente Pro, let us know. If you want to get notified, when AlDente Pro comes out, just enter your email at https://apphousekitchen.com/aldente/ In the meantime, stay safe and keep your battery fresh. David & Matthias
